src/backend/emails/_footer.html.ejs

```diff
@@ -0,0 +1,31 @@
+                </td>
+              </tr>
+
+              <!-- END MAIN CONTENT AREA -->
+              </table>
+
+            <!-- START FOOTER -->
+            <div class="footer" style="clear: both; padding-top: 24px; padding-bottom: 24px; text-align: center; width: 100%;">
+              <table role="presentation" border="0" cellpadding="0" cellspacing="0" style="border-collapse: separate; mso-table-lspace: 0pt; mso-table-rspace: 0pt; width: 100%;" width="100%">
+                <tr>
+                  <td class="content-block" style="font-family: Helvetica, sans-serif; vertical-align: top; color: #9a9ea6; font-size: 16px; text-align: center;" valign="top" align="center">
+                    <span class="apple-link" style="color: #9a9ea6; font-size: 16px; text-align: center;">Copyright &copy; <%= new Date().getFullYear() %> Compass</span>
+                  </td>
+                </tr>
+                <tr>
+                  <td class="content-block powered-by" style="font-family: Helvetica, sans-serif; vertical-align: top; color: #9a9ea6; font-size: 16px; text-align: center;" valign="top" align="center">
+                    <a href="<%= env.BASE_URL %>" style="color: #9a9ea6; font-size: 16px; text-align: center; text-decoration: none;"><%= env.BASE_URL %></a>
```

Comment:
do we want to import just baseURL as a variable, rather than the entire `env`?

Reply:
I've updated the mailer helper to add only a subset of the env into the mail template view options.